packages/editor-preview/src/browser/editor-preview-manager.ts

```diff
@@ -122,4 +133,28 @@ export class EditorPreviewManager extends EditorManager {
             widget.convertToNonPreview();
         }
     }
+
+    protected override async revealSelection(widget: EditorWidget, options: EditorOpenerOptions = {}, uri?: URI): Promise<void> {
+        if (!options.selection) {
+            options.selection = await this.resolveSelection(options, widget, uri);
+        }
+        super.revealSelection(widget, options, uri);
+    }
+
+    protected async resolveSelection(options: EditorOpenerOptions, widget: EditorWidget, uri: URI | undefined): Promise<RecursivePartial<Range> | undefined> {
+        if (!options.selection) {
```

Comment:
This was already checked before in `revealSelection`, so no need to check it again.

Reply:
Right, but this is not reflected in the parameters, so I wouldn't rely on this information. Especially as this could be called by subclasses.